R/chromote_session.R

```diff
@@ -14,25 +23,16 @@ ChromoteSession <- R6Class(
       parent = default_chromote_object(),
       width = 992,
       height = 744,
+      tid = create_blank_target(parent, width, height)$targetId,
```

Comment:
I'd prefer it if this defaulted to `NULL`. If NULL, create a new target (like the current behavior); if not NULL, use that `tid` instead.

With this function call as the default value, it's hard for users to understand what it's doing, especially since the function isn't exported.

Also, does it work as written? I'd expect `create_blank_target()` to return a promise, which you can't get `$targetId` from.

Reply:
It didn't work, but it does now 😄 

After this comment it occurred to me that maybe we should take a Target instead of just the targetId, but I poked around the devtools APIs and it looks like everything parameterized by a Target takes just a targetId. 👍 

To the extent alignment with the underlying devtools API is helpful, I wonder if it would be a good idea to call our new parameter `targetId` instead of `tid`?

Reply:
I think you're right that `targetId` is preferable.